Library/Homebrew/extend/os/linux/extend/ENV/super.rb

```diff
@@ -20,4 +20,25 @@ def determine_rpath_paths
     paths += homebrew_extra_library_paths
     paths.to_path_s
   end
+
+  def x11
+    xorg = Formula["linuxbrew/xorg/xorg"]
+    if xorg.installed?
+      xorg.recursive_dependencies.each do |d|
+        dep_formula = d.to_formula
+        append_path "PKG_CONFIG_LIBDIR", dep_formula.lib/"pkgconfig"
+        append_path "PKG_CONFIG_LIBDIR", dep_formula.share/"pkgconfig"
+
+        append "LDFLAGS", "-L#{dep_formula.lib}"
+        append_path "CMAKE_PREFIX_PATH", dep_formula.prefix.to_s
+        append_path "CMAKE_INCLUDE_PATH", dep_formula.include.to_s
+
+        append "CPPFLAGS", "-I#{dep_formula.include}"
+
+        append_path "ACLOCAL_PATH", dep_formula.share/"aclocal"
```

Comment:
For `--env=super`, these standard environment variables are all empty. Take a look at how `extend/os/mac/extend/ENV/super.rb` handles `x11`. Grep for ` if x11?`